### PR TITLE
portal: switching to go-azure-sdk

### PR DESCRIPTION
**Pull request by stephybun**

### CODE REVIEW

File: internal/services/portal/client/client.go [hunk 1]
```diff
@@ -2,23 +2,30 @@ package client
 
 import (
 	"github.com/Azure/azure-sdk-for-go/services/preview/portal/mgmt/2019-01-01-preview/portal"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/portal/2019-01-01-preview/dashboard"
```
Comment (tombuildsstuff) [inline internal/services/portal/client/client.go:5]: `go mod vendor`?

Comment (stephybun) [reply internal/services/portal/client/client.go:5]: whoops, thanks tom 🙂

**Review by tombuildsstuff — APPROVED** [review]
LGTM 👍

**Comment by tombuildsstuff** [comment]
Tests look good 👍

**Pull request merged**